### DISCUSSION TITLE
关于加载预训练模型的问题

### DISCUSSION THREAD

**Discussion by ayicherry** [open-mmlab/mmclassification#1200] [General] [answered]
加载预训练模型好像有两个地方可以设置，第一个就是在模型设置的参数里，有个checkpoint的设置，type是pretrained，这个在模型初始化的时候会加载。然后第二个地方是运行设置参数里有个load_from参数，也会加载模型参数，请问这两个有什么区别？看顺序好像是模型设置那个先加载，然后运行设置里的后加载，会覆盖？

**Comment by mzr1996** [comment, marked as answer]
这里有对比
https://mmclassification.readthedocs.io/zh_CN/latest/tutorials/runtime.html#resume-from-load-from-init-cfg-pretrained
